### PR TITLE
feat(server): init locks statuses

### PR DESCRIPTION
**Pull request by ArturMichalak**
## Description

Each newly created vehicle has its initial lock status set. This is a one-time action for each vehicle.

## Checklist

- [X] I have personally loaded this code into an updated Qbox project and checked all of its functionality.
- [X] My pull request fits the contribution guidelines & code conventions.

### CODE REVIEW

**Comment by ArturMichalak** [comment]
I developed the mechazm while working on my own implementation of vehiclekeys.
I would like to invite you to analyze the solution.

File: server/main.lua [hunk 1]
```diff
@@ -49,3 +52,13 @@ end)
...
     removePlayer(source --[[@as integer]])
 end)
+
+---Lock every spawned vehicle
+---@param vehicle number The entity number of the vehicle.
+AddEventHandler('entityCreated', function (vehicle)
+    if not vehicle or GetEntityType(vehicle) ~= 2 or GetEntityPopulationType(vehicle) > 5 then return end
+    local isDriver = GetPedInVehicleSeat(vehicle, -1) ~= 0
+    local isLocked = (config.lockNPCDrivenCars and isDriver) or (config.lockNPCParkedCars and not isDriver)
+                        and GetVehicleType(vehicle) ~= 'bike' and not getIsVehicleBlacklisted(vehicle)
```
Comment (Manason) [inline server/main.lua:62]: We should rename getIsVehicleBlacklisted. I think it's a little unclear what the blacklist means whether it's always locked, or always unlocked so we should just name it the spawnedLocked or spawnedUnlocked list to be more clear.

**Review by Manason — APPROVED** [review]

File: server/main.lua (hunk 1 above)
Comment (ArturMichalak) [reply server/main.lua:62]: getIsVehicleAlwaysUnlocked?

Comment (Manason) [reply server/main.lua]: I guess. It's not always unlocked though, it just spawns unlocked?

Comment (ArturMichalak) [reply server/main.lua]: This must also be used when toggle the lock

**Pull request merged**